src/js/pages/Theme.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { withRouteData } from 'react-static';
+import PropTypes from 'prop-types';
+
+// TODO: this jsonFileData is temporary. Add it to Wagtail API
+import jsonFileData from '__tmpdata/pages';
+import TileGroupSet from 'js/modules/TileGroupSet';
+import PageBreadcrumbs from 'js/modules/PageBreadcrumbs';
```

Comment:
kinda weird that we don't use breadcrumbs, but since we don't, you can kill the import.